### DISCUSSION TITLE
Next planned release estimated date

### DISCUSSION THREAD

**Discussion by rustamhajiyev** [Eventuous/eventuous#234] [General]
Hello @alexeyzimarev and contributors  🙂 I plan to start using this awesome library in my next project but wondering if I should use self compiled version from main branch, since the latest release was in February or maybe I should wait a bit till you release a new version?

Sorry if it's a duplicate, I did some searching but couldn't find any info about the roadmap.

Thanks!

**Comment by alexeyzimarev** [comment]
The main branch is published as preview packages available on MyGet. I believe it's mentioned in README. The current trunk contains some breaking changes, so I have to do some cleanup work before releasing 0.15.0. I need to merge the remaining PR, and see if there are some small fixes that I could do quickly, then I'll publish a beta.

More of my time currently goes to connectors, so I am a bit behind schedule on the main library.

**Reply by rustamhajiyev** [reply]
Thank you @alexeyzimarev for the detailed answer :)
